src/name_bazaar/ui/components/offering/list_item.cljs

```diff
@@ -59,6 +71,8 @@
       {:centered true}
       [offering-middle-section
        {:offering offering}]]
+     (when (not (and valid-name? normalized?))
```

Comment:
This whole part including `non-valid-name-warning` function should be in `name-bazaar.ui.components.offering.middle-section` with other warnings